simulation/vis_simulate.pro

```diff
@@ -49,9 +51,9 @@ FUNCTION vis_simulate,obs,status_str,psf,params,jones,skymodel,file_path_fhd=fil
       ENDELSE
 
     endfor
-    IF ~Keyword_Set(no_save) THEN save, file=init_beam_filepath, beam2_xx_image, beam2_yy_image, obs
-    undefine_fhd, beam2_xx_image, beam2_yy_image,beam_arr
-    
+;    IF ~Keyword_Set(no_save) THEN save, file=init_beam_filepath, beam2_xx_image, beam2_yy_image, obs
```

Comment:
I can't see a reason to remove this line

Reply:
The saved beam models were hogging up a lot of storage for me, so I cut the line since I didn't need them. :)
I'll put it back.